app/controllers/users/passwords_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Users::PasswordsController < Devise::PasswordsController
+  # GET /resource/password/new
+  # def new
+  #   super
+  # end
+
+  # POST /resource/password
+  # def create
+  #   super
+  # end
+
+  # GET /resource/password/edit?reset_password_token=abcdef
```

Comment:
@Br3nda there is a checkbox for password reset in the review template, but I haven't tested this yet.